README.md

```diff
@@ -101,6 +101,22 @@ then this module is intended for you.
 [thinking-in-react]: https://reactjs.org/docs/thinking-in-react.html
 [bootstrap]: https://getbootstrap.com
 
+## Alternatives
+
+The main drawback of this API is that it is somewhat implicit and relies on the components
+having the right `displayName`. This couples the "layout" components and the "child" components
+[very tightly](https://github.com/jxnblk/macro-components/issues/3#issuecomment-361997713)
+because a change in the API of the "layout" component will require changes in
+the source of the child components passed to it.
+
+This API also doesn't work if the component relies on a `name`
```

Comment:
I think I *totally* get the issue here now – by using the `name` prop for both placement within a macro-component **and** for a form element's name, you'd need to do nasty things for that to work as expected. I added the `name` prop as a secondary/fallback sort of thing, but it's certainly not how I'd want this whole thing to work generally. I might get rid of that altogether but would definitely be a breaking change